src/resolve/fuzz-etc-hosts.c

```diff
@@ -11,6 +11,9 @@ int LLVMFuzzerTestOneInput(const uint8_t *data, size_t size) {
         if (!getenv("SYSTEMD_LOG_LEVEL"))
                 log_set_max_level(LOG_CRIT);
 
+        if (size > 64*1024)
```

Comment:
`if (HAS_FEATURE_ADDRESS_SANITIZER && size > …)` ?

Reply:
I'm not *convinced* that this is the way to go. I just don't want to disable the test too much.